topics/list_topics.go

```diff
@@ -0,0 +1,35 @@
+package topics
+
+import (
+	"context"
+	"regexp"
+
+	kafkago "github.com/segmentio/kafka-go"
+)
+
+// ListTopics returns a slice of all the Topics
+func ListTopics(ctx context.Context, client *kafkago.Client) (topics []kafkago.Topic, err error) {
+	response, err := client.Metadata(ctx, &kafkago.MetadataRequest{
+		Addr: client.Addr,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return response.Topics, nil
+}
+
+// ListTopicsRegex returns a slice of Topics that match a regex
+func ListTopicsRegex(ctx context.Context, client *kafkago.Client, match regexp.Regexp) (topics []string, err error) {
```

Comment:
Mostly aesthetic comments, but the functional change here is the `*` in the `regexp.Regexp`
```suggestion
func ListTopicsRegex(ctx context.Context, cli *kafkago.Client, re *regexp.Regexp) (topics []string, err error) {
```

Reply:
Done